multiarch/generate-pipeline.groovy

```diff
@@ -53,7 +53,7 @@ node {
 		def archImages = [:]
 
 		// https://github.com/docker-library/meta/blob/subset/subset.txt
-		def subset = ['docker', 'notary', 'rabbitmq']
+		def subset = ['docker', 'notary', 'rabbitmq', 'postgres']
```

Comment:
We'll probably be adding more over time; can you add it in sorted order?

Reply:
is there a way to make this multiline too?

Reply:
Updated and switched to multiline